### PR TITLE
Hide Casino Games

### LINKED ISSUE #117: Hide Casino Games

### PR DESCRIPTION
**Pull request by Sashank999**
Closes #117 .

### CODE REVIEW

File: extension/scripts/features/hidecasinogames/ttHideCasinoGames.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		unhideCasinoGames,
+		{
+			storage: ["settings.pages.sidebar.collapseAreas"],
+		},
+		async () => {
+			if (await checkMobile()) return "Not supported on mobile!";
+
+			await requireSidebar();
+
+			if (document.find("#sidebarroot .tablet")) return "Already collapsible.";
```
Comment (DeKleineKobini) [inline extension/scripts/features/hidecasinogames/ttHideCasinoGames.js:19]: You copied a bit too much xD

File: extension/scripts/global/globalData.js [hunk 2]
```diff
@@ -214,6 +215,7 @@ const DEFAULT_STORAGE = {
 				highlights: new DefaultSetting({ type: "array", defaultValue: [{ name: "$player", color: "#7ca900" }] }),
 				titleHighlights: new DefaultSetting({ type: "array", defaultValue: [] }),
 			},
+			casino: {},
```
Comment (DeKleineKobini) [inline extension/scripts/global/globalData.js:218]: For an object as type you need to put `new DefaultSetting({ type: "object", defaultValue: {} })`

**Review by DeKleineKobini — CHANGES_REQUESTED** [review]

File: extension/scripts/features/hidecasinogames/ttHideCasinoGames.js (hunk 1 above)
Comment (Sashank999) [reply extension/scripts/features/hidecasinogames/ttHideCasinoGames.js]: lol removed it.
It is supported on mobile. Tested it in chrome emulation.

File: extension/scripts/global/globalData.js (hunk 2 above)
Comment (Sashank999) [reply extension/scripts/global/globalData.js]: I actually don't need that so I left it as a place for future options. Removed it.

**Pull request merged**